agent/bench-scripts/pbench-cyclictest

```diff
@@ -160,6 +160,7 @@ done
 if [ $stress ]; then
 	kill $stress_pid
 fi
+pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir
```

Comment:
Add "`end`" argument, and invoke it with the "`beg`" argument between lines 126 and 127.